diagram-viewer/src/main/java/com/powsybl/diagram/viewer/sld/SingleLineDiagramModel.java

```diff
@@ -312,17 +306,51 @@ public SubstationLayoutFactory fromString(String item) {
         };
     }
 
-    public StringConverter<VoltageLevelLayoutFactoryCreator> getVoltageLevelLayoutFactoryCreatorStringConverter() {
+    public StringConverter<VoltageLevelLayoutFactoryBean.VoltageLevelLayoutFactoryType> getVoltageLevelLayoutFactoryCreatorStringConverter() {
```

Comment:
There's no need for a converter, you can remove this. If you want to display something else than the enum `name()`, you can override the `toString` method in the enum.